cli/src/alluxio.org/cli/env/service.go

```diff
@@ -0,0 +1,57 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package env
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var serviceRegistry = map[string]*Service{}
+
+func RegisterService(p *Service) *Service {
```

Comment:
Service may be ambiguous with the Overmind naming. [These docs](https://primer.style/design/native/cli/foundations) use the word "object", but we could come up with something else if we wanted.

Reply:
🙄